src/Apps/Home/Components/HomeContentCards.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react"
+import { HeroCarousel } from "Components/HeroCarousel/HeroCarousel"
+import {
+  HomeHeroUnit,
+  StaticHeroUnit,
+} from "Apps/Home/Components/HomeHeroUnits/HomeHeroUnit"
+
+interface ContentCardProps {
+  card: any
+  index: any
```

Comment:
Can we type these?

Reply:
omg I totally forgot about this thank you - will fix!

Reply:
For those Braze types do I just do something like `yarn add -D @braze/web-sdk`? And then I can import them? And that won't make it into the prod bundle because of it being a dev dep?